tests/unittests/utils_tests.py

```diff
@@ -463,12 +473,19 @@ def test_multiple_config_cleanup(self):
             }
         )
 
-        # Ensure that duplicate documents are automatically cleaned up
         config = foo.get_db_config()
 
-        self.assertEqual(len(list(db.config.aggregate([]))), 1)
+        if fo.config.database_admin:
+            # Ensure that duplicate documents are automatically cleaned up if run by database admin
+            self.assertEqual(len(list(db.config.aggregate([]))), 1)
+        else:
+            # Otherwise, the duplicates are not cleaned up
+            self.assertEqual(len(list(db.config.aggregate([]))), 3)
         self.assertEqual(config, orig_config)
 
+        # Clean up the fake configs
+        db.config.delete_many({"_id": {"$in": new_config_ids}})
```

Comment:
put this in a try-finally block so they'll still be deleted even if an assertion or other call fails

Reply:
good idea